cores/arduino/board.h

```diff
@@ -3,14 +3,23 @@
 
 /*
  * Core and peripherals registers definitions
- */
+*/
+#ifdef __cplusplus
+extern "C"{
+#endif // __cplusplus
 #include "analog.h"
 #include "clock.h"
 #include "core_callback.h"
 #include "digital_io.h"
 #include "hal_uart_emul.h"
 #include "hw_config.h"
+#ifdef __cplusplus
+}
+#endif // __cplusplus
 #include "interrupt.h"
```

Comment:
Could you move include  before to avoid multiple 
#ifdef __cplusplus
extern "C"{
#endif // __cplusplus